gryphon/lib/analysis/legacy/volatility.py

```diff
@@ -346,8 +346,8 @@ def volatility(values, timestamps, window_time=10000):
             else:
                 break
         
-        standard_deviation = np.std(relevant_values)
-        mean = np.mean(relevant_values)
+        standard_deviation = numpy.std(relevant_values)
+        mean = numpy.mean(relevant_values)
```

Comment:
It's a convention among Python developers to use `np` for numpy. It will be understood by everybody :)

Reply:
Yes but it is not defined in this file -- there is no __import numpy as np__ in this file.

Instead, in this file there is only __inport numpy_ and several existing calls to __numpy.__xyz()

Reply:
my bad, didn't notice the missing import :)

Reply:
flake8 is awesome for catching things like this that we humans so easily tend to miss.

Reply:
This code is honestly deprecated, it could probably be removed entirely. That's likely why there was a syntax error in there.